lib/screens/profile/contact_support.dart

```diff
@@ -197,23 +189,23 @@ class _ContactSupportState extends State<ContactSupport> {
                     fontSize: 16,
                   ),
                 ),
-                DeviceSpecList(deviceInfos: deviceInfos),
+                _DeviceInfo(deviceInfos: deviceInfos()),
               ] else
                 FutureBuilder<String>(
                   future: ajaxCall,
                   builder:
                       (BuildContext context, AsyncSnapshot<String> snapshot) {
-                    Widget returnWidget;
+                    // Widget returnWidget;
                     if (snapshot.hasData) {
-                      returnWidget = SendingResultPage(
-                        sendingResult: false,
+                      return SendingResultPage(
```

Comment:
SendingResultPage -> _SendingResultPage. Do this for all except the primary widget for the file.